frappe/core/doctype/server_script/server_script_utils.py

```diff
@@ -22,9 +22,12 @@
 	"before_update_after_submit": "Before Save (Submitted Document)",
 	"on_update_after_submit": "After Save (Submitted Document)",
 	"before_print": "Before Print",
-	"on_payment_paid": "On Payment Paid",
 	"on_payment_failed": "On Payment Failed",
-	"on_payment_authorized": "On Payment Authorization",
+	"on_payment_charge_processed": "On Payment Charge Processed",
+	"on_payment_mandated_charge_processed": "On Payment Mandate Charge Processed",
+	"on_payment_mandate_acquisition_processed": "On Payment Mandate Acquisition Processed",
+	"on_payment_paid": "On Payment Paid (Legacy Gateway)",
+	"on_payment_authorized": "On Payment Authorization (Legacy Gateway)",
```

Comment:
I'm not too familiar with this, would this break anything in terms of backwards compatibility?

Reply:
Good point, maybe it's not yet the right time to mark them as legacy (just yet). I'll revert that part of the change, once they become legacy, this would probably require a migration.

Reply:
I removed the legacy marking for now: https://github.com/frappe/frappe/pull/25952/commits/d1469fde66efa3032035cde95ed26dabb459de6e